web/core/build.gradle.kts

```diff
@@ -30,7 +30,6 @@ kotlin {
         val jsMain by getting {
             dependencies {
                 implementation(kotlin("stdlib-js"))
-                implementation(npm("css-typed-om", "0.4.0"))
```

Comment:
Can you please remove css-typed-om license from licenses as well?

Reply:
Sure!